bob/src/hw_metrics_collector.rs

```diff
@@ -69,6 +70,8 @@ impl HWMetricsCollector {
             debug!("used mem in mb: {}", used_mem);
             gauge!(USED_RAM, used_mem as f64);
             gauge!(FREE_RAM, (total_mem - used_mem) as f64);
+            let bob_ram = bytes_to_mb(Self::bob_ram());
```

Comment:
here is no reason to get RAM for a process directly from procfs. The `Sysinfo` crate already has the ability to get process memory. 

Also, please, review the `sys.refresh_*` calls above. I think `refresh_all` is not needed, it is better to refresh only required parts